File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/02_Easy/12157_:::_Tariff_Plan/Main.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main
{
	public static void main(String[] args) throws IOException
	{
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
		//-----------------------------------------------------------
		int casos=Integer.parseInt(br.readLine()), N, mile, juice;
		double estaCall;
		String aux[], rta="";
		
		for(int c=1; c<=casos; c++)
		{
			mile=juice=0;
			N=Integer.parseInt(br.readLine());
			aux=br.readLine().split("\\s+");
			
			for(int i=0; i<N; i++) 
			{
				estaCall=Double.parseDouble(aux[i]);
				mile +=Math.ceil(estaCall/30.0)*10+((estaCall%30==0)?10:0);
				juice+=Math.ceil(estaCall/60.0)*15+((estaCall%60==0)?15:0);
			}
			
			rta = (mile<juice)?"Mile "+mile : (mile>juice)?"Juice "+juice :"Mile Juice "+mile;
			System.out.println("Case "+c+": "+rta);
		}
		//-----------------------------------------------------------
		br.close();
	}
}

File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/02_Easy/12157_:::_Tariff_Plan/Main.cpp
using namespace std;
#include <cstdio>
#include <math.h>

int main()
{
	int casos, N, mile, juice, c, i;
	double estaCall;
	scanf("%d", &casos);

	for(c=1; c<=casos; c++)
	{
		mile=juice=0;
		scanf("%d", &N);

		for(i=0; i<N; i++)
		{
			scanf("%lf", &estaCall);
			mile +=ceil(estaCall/30.0)*10+((fmod(estaCall, 30.0)==0)?10:0);
			juice+=ceil(estaCall/60.0)*15+((fmod(estaCall, 60.0)==0)?15:0);
		}

		printf("Case %d: %s %d\n", c, (mile<juice)?"Mile":(mile>juice)?"Juice":"Mile Juice", (mile<=juice)?mile:juice);
	}

	return 0;
}
